src/common/i18n/locales/en-US.json

```diff
@@ -200,7 +200,12 @@
     "buy-account-li-1": "Instant creation without extra checks and waiting",
     "buy-account-li-2": "3x RC compared to normal new accounts",
     "buy-account-li-3": "300 Points to help you get started",
-    "qr-desc": "Scan with Ecency mobile app to continue"
+    "qr-desc": "Scan with Ecency mobile app to continue",
+    "username-min-length-error": "Length should be at least 3 symbol",
+    "username-max-length-error": "Length should be no more than 16 symbols",
+    "username-no-ascii-first-letter-error": "First letter should be in ASCII",
+    "username-contains-symbols-error": "Should contains letters, numbers, hyphen only",
```

Comment:
@dkildar, can contain dot as well, only each side of dot should be longer than 3 characters, iirc

Reply:
@dkildar also check these, this is from blockchain source code, much more accurate (apparently this type of username `xyz12` also not supported): https://gitlab.syncad.com/hive/hive/-/blob/master/libraries/protocol/include/hive/protocol/authority.hpp#L76